spec/api/read_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+describe "app" do
+  describe "read" do
+
+    before(:each) do
+      set_api_key_header
+      setup_10_threads
+    end
+
+    describe "POST /api/v1/users/:user_id/read" do
+      it "marks a thread as read for the user" do
+        thread = @threads["t0"]
+        user = create_test_user(42)
+        current_time = Time.now.utc.strftime("%Y-%m-%dT%H:%M:%SZ")
+        post "/api/v1/users/#{user.external_id}/read", source_type: "thread", source_id: thread.id
+        last_response.should be_ok
+        user.reload
+        read_state = user.read_states.find_or_create_by(course_id: thread.course_id)
+        read_state.last_read_times[thread.id.to_s].utc.strftime("%Y-%m-%dT%H:%M:%SZ").should == current_time
```

Comment:
@jcdyer I looked a little more at existing behaviour and noticed a common way of asserting if thread is marked as read and have updated the test according to it. It is even simpler than mocking the date.
For refernce; https://github.com/edx/cs_comments_service/blob/jia/MA-2419/spec/spec_helper.rb#L221